clients/admin-ui/src/pages/add-systems/multiple.tsx

```diff
@@ -28,10 +29,10 @@ const AddMultipleSystemsPage: NextPage = () => (
     <Box w={{ base: "100%", md: "75%" }}>
       <Text fontSize="sm" mb={8}>
         {DESCRIBE_SYSTEM_COPY}
-        <Link href={ADD_SYSTEMS_MANUAL_ROUTE} color="complimentary.500">
-          {" "}
-          click here.{" "}
-        </Link>
+        <NextLink href={ADD_SYSTEMS_MANUAL_ROUTE} passHref legacyBehavior>
```

Comment:
I've had to do this before, I don't understand how we get to a world where we don't need `legacyBehavior`

Reply:
It's not something I worry about too much. It's basically the means to pass href to the component inside it. I used it a lot with MUI and we need to use it with Ant. I think `legacyBehavior` is just a scary misnomer.